jamovi/surv.a.yaml

```diff
@@ -71,4 +66,24 @@ options:
       title: Censored events
       type: Bool
       default: false
+  
+    - name: risktable
```

Comment:
just call this one `risk` ... we don't normally include `table` or `image` on the end of names

Reply:
Ok. I made the change